canonical-web-and-design/ubuntu.com
# fix: use side navigation instead of sticky table of contents

## Done

- This feature add accordions vanilla-framework to table of contents sticky.

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- go to
     - http://127.0.0.1:8001/legal/ubuntu-advantage-service-description
     - http://127.0.0.1:8001/legal/ubuntu-advantage-service-description/ja

## Issue / Card
Fixes #10838 

## Screenshots

![toc-accordions](https://user-images.githubusercontent.com/20561190/141656023-06561038-d676-4e56-9827-45746f8e5864.gif)


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
